wcfsetup/install/files/lib/core.functions.php

```diff
@@ -710,6 +715,36 @@ function sanitizeStacktrace(\Throwable $e, bool $ignorePaths = false) {
 			if (!isset($item['class'])) $item['class'] = '';
 			if (!isset($item['type'])) $item['type'] = '';
 			if (!isset($item['args'])) $item['args'] = [];
+
+			if ($item['class']) {
+				$f = new \ReflectionMethod($item['class'], $item['function']);
```

Comment:
Why `$f`, can it be at least something like `$func` instead?

Reply:
Remnants from my initial PoC implementation :smile: